docs/2.7.0/docs/daml-finance/tutorials/settlement/internal.rst

```diff
@@ -0,0 +1,153 @@
+Bank, and the requestor finally settles the batch:
+
+.. literalinclude:: ../../finance-settlement/daml/Scripts/Internal.daml
+  :language: daml
+  :start-after: -- WRAPPED_TRANSFER_SETTLE_BEGIN
+  :end-before: -- WRAPPED_TRANSFER_SETTLE_END
+
+Note that this occurs without involving the Bank, and either Alice or Bob could also take the role
+as the requestor. As a result of running this script, Alice's holding is transferred to Bob.
+
+Credit and Debit
+================
+
+An alternative approach to transfer the holding from Alice to Bob includes the Bank as an
+intermediary.
+
+.. literalinclude:: ../../finance-settlement/daml/Scripts/Internal.daml
+  :language: daml
+  :start-after: -- CREDIT_DEBIT_INSTRUCT_BEGIN
+  :end-before: -- CREDIT_DEBIT_INSTRUCT_END
+
+Similar to the previous scenario, Alice allocates by pledging a holding, and Bob approves by taking
+delivery to his account. However, in this case, the Bank plays an intermediary role by allocating
+and approving, debiting the sender and crediting the receiver, respectively:
+
+.. literalinclude:: ../../finance-settlement/daml/Scripts/Internal.daml
+  :language: daml
+  :start-after: -- CREDIT_DEBIT_SETTLE_BEGIN
+  :end-before: -- CREDIT_DEBIT_SETTLE_END
+
+We could have made the Bank approve and allocate its instructions by taking delivery to an account
+it owns and pledging a holding where it acts as the custodian. However, this would necessitate
+"dummy" accounts and holdings, which can be avoided using the ``DebitSender`` and ``CreditReceiver``
+methods. These methods are exclusively accessible to the custodian.
+
+Pass Through
+============
+
+The final script of this tutorial demonstrates how holdings received in the same settlement process
+can be allocated to a subsequent instruction. We again use two instructions, ``instruction1`` and
+``instruction2``, but now with Charlie as the intermediary:
+
+.. literalinclude:: ../../finance-settlement/daml/Scripts/Internal.daml
+  :language: daml
+  :start-after: -- PASS_THROUGH_INSTRUCT_BEGIN
+  :end-before: -- PASS_THROUGH_INSTRUCT_END
+
+As in previous examples, Alice allocates by pledging a holding, and Bob approves by taking delivery
+to his account. The intermediary, Charlie, allocates using
+``PassThroughTo (charlieAccount, instruction2)`` and approves with
+``PassThroughFrom (charlieAccount, instruction1)``, essentially enabling the holding Charlie
+receives from Alice to pass-through to Bob:
+
+.. literalinclude:: ../../finance-settlement/daml/Scripts/Internal.daml
+  :language: daml
+  :start-after: -- PASS_THROUGH_SETTLE_BEGIN
+  :end-before: -- PASS_THROUGH_SETTLE_END
+
+The significant advantage of the pass-through method is that Charlie doesn't need any holdings
+upfront as he's at a net zero position for incoming and outgoing holdings in this settlement process.
+
+Note that the Bank could have utilized the pass-through approach to achieve the same result, but it
+would still require a "dummy" account.
+
+Run the Script
```

Comment:
I would put this section before you explain the scripts, or remove it altogether

Reply:
I removed it.